### ISSUE TITLE
Restoring NuGet packages listed in packages.config seem broken in Nuget restore version 2

### ISSUE THREAD

**Issue by mohammadtibi** [jfrog/artifactory-azure-devops-extension#151]
When using the build task Artifactory Nuget restore version 2.*  with a packages.config file and we provide a destination directory or specify the complete path to the file in this field "Path to the root directory of the solution or to a file inside the root directory". - The restore command fails with the below exception
```
Cannot determine the packages folder to restore NuGet packages. Please specify either -PackagesDirectory or -SolutionDirectory.
System.InvalidOperationException: Cannot determine the packages folder to restore NuGet packages. Please specify either -PackagesDirectory or -SolutionDirectory.
   at NuGet.CommandLine.RestoreCommand.GetPackagesFolder(PackageRestoreInputs packageRestoreInputs)
   at NuGet.CommandLine.RestoreCommand.<PerformNuGetV2RestoreAsync>d__40.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.CommandLine.RestoreCommand.<ExecuteCommandAsync>d__34.MoveNext()
```

While in version 1, it works as expected.

It seems like when working with version 1, we send the below command with the JFrog CLI:

`C:\hostedtoolcache\windows\jfrog\1.35.3\x64\jfrog.exe rt nuget restore docker --solution-root="D:\a\1\s\nuget-example\multi-example\core" --nuget-args="-NoCache -PackagesDirectory D:\\a\\1\\s -Verbosity Detailed" --url="<ART-URL>" --user=*** --password=***`

While with version 2 when we fail, we only send the below command without any arguments:
`C:\hostedtoolcache\windows\jfrog\1.35.3\x64\jfrog.exe rt nuget restore -NoCache -Verbosity Detailed
`

It seems like if we manually pass the `-PackagesDirectory ` argument with version 2, this fixes the issue.
It could be that the issue is not sending the complete arguments with the CLI when working with version 2

**Comment by Or-Gabay** [comment]
Thanks for reporting this issue @mohammadtibi,
The 'core' folder contains ''packages.config'', therefore NuGet requires a valid path to store the installed dependencies, one way is to supply the flag 'PackagesDirectory'.
I will add this flag to version 2: https://github.com/jfrog/artifactory-azure-devops-extension/pull/152

**Comment by Or-Gabay** [comment]
@mohammadtibi, we released Artifactory Azure DevOps Extension v1.10.0 to resolve this issue.
